icu4c/source/test/testdata/Burmese_graphclust_model5_heavy.txt

```diff
@@ -1,4 +1,4 @@
-// © 2021 and later: Unicode, Inc. and others.
+﻿// © 2021 and later: Unicode, Inc. and others.
```

Comment:
Can/should you also now remove the `-eUTF-8` that you added in #1689?

Reply:
ok, trying.